src/cocoa/toga_cocoa/widgets/table.py

```diff
@@ -18,12 +18,14 @@ def tableView_objectValueForTableColumn_row_(self, table, column, row: int):
     # TableDelegate methods
     @objc_method
     def tableViewSelectionDidChange_(self, notification) -> None:
-        print ("selection changed")
-
+        print ("selection changed to row: %s" % notification.object.selectedRow)
```

Comment:
nit: I know this already existed, but I for one would prefer to not have this print statement lying around at all.